src/SFA.DAS.EmployerAccounts.Web/Startup.cs

```diff
@@ -214,6 +214,9 @@ private Task OnSecurityTokenValidated(SecurityTokenValidatedNotification<WsFeder
                 }
                 else if (notification.AuthenticationTicket.Identity.HasClaim(serviceClaimType, "ESS"))
                 {
+                    Logger.Debug("Adding Tier1 Role");
```

Comment:
I don't think we need to add this additional claim. Tier2 users will end up with 3 Role claims one of which being Tier1User. Let's just use Tier2User and ConsoleUser for now.